spiketools/stats/anova.py

```diff
@@ -49,17 +50,21 @@ def create_dataframe(data, columns=None, drop_na=True, types=None):
     return df
 
 
-def create_dataframe_bins(data, columns, drop_na=True):
+def create_dataframe_bins(bin_data, columns=None, other_data=None, drop_na=True):
     """Create a dataframe from an array of binned data.
 
     Parameters
     ----------
-    data : 2d or 3d array
+    bin_data : 2d or 3d array
         An array of data organized into pre-computed bins.
         If a 2d array, should be organized as [n_trials, n_bins].
         If a 3d array, should be organized as [n_trials, n_xbins, n_ybins].
-    columns : list of str
-        The column labels for the dataframe.
+    columns : list of str, optional
+        The column labels for the bin data.
+        Defaults to ['bin', 'fr'] for 1d bins or ['xbin', 'ybin' 'fr'] for 2d bins.
+    other_data : dict
```

Comment:
Should `other_data` be optional too?

Reply:
yes! I keep forgetting optional labels, thanks for finding!